cc.config.json

```diff
@@ -2,15 +2,64 @@
     "$schema": "./cc.config.schema.json",
 
     "features": {
-        "base": { "modules": [ "base", "native" ], "dependentAssets": ["0835f102-5471-47a3-9a76-01c07ac9cdb2", "b5475517-23b9-4873-bc1a-968d96616081", "0ed97c56-390e-4dd1-96b7-e7f2d93a98ed", "b23391b6-52eb-46a6-8da1-6244d9d315fb"] },
+        "base": { "modules": [ "base", "native" ],  "dependentAssets": [
+            "b23391b6-52eb-46a6-8da1-6244d9d315fb", 
+            "b5475517-23b9-4873-bc1a-968d96616081", 
+            "0ed97c56-390e-4dd1-96b7-e7f2d93a98ed", 
+            "0835f102-5471-47a3-9a76-01c07ac9cdb2",
+            "ff9be190-20a4-4e48-b68c-76e3c7cff085",
```

Comment:
Where are these UUID come from? 
Select a file in asset window and show UUID ?

Reply:
Yeah, you can see uuid in editor or meta.file